app/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -571,7 +571,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/DebugProfile.entitlements;
-				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```

Comment:
TODO: Get rid of this
cargo checkout origin/main app/macos/Runner.xcodeproj/project.pbxproj